adapters/nsw.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+var request = require('request');
+var _ = require('lodash');
+var moment = require('moment-timezone');
+var cheerio = require('cheerio');
+
+exports.name = 'nsw';
+
+exports.fetchData = function (source, cb) {
+  request(source.url, function (err, res, body) {
+    if (err || res.statusCode !== 200) {
+      console.error(err || res);
+      return cb({message: 'Failure to load data url.'});
+    }
+
+    // Wrap everything in a try/catch in case something goes wrong
+    try {
+      // Format the data
+      var data = formatData(body);
+      // Make sure the data is valid
+      if (data === undefined) {
+        return cb({message: 'Failure to parse data.'});
+      }
+      cb(null, data);
+    } catch (e) {
+      return cb({message: 'Unknown adapter error.'});
+    }
+  });
+};
+
+// This is pretty nasty, but the rest of the code is as well.
+var parseDate = function (dateString) {
+  var d = dateString.split('<br>');
+  // Split the hour string to get starting hour + am/pm
+  var h = d[2].split(' ');
+  var date = moment.tz(d[1] + h[0] + h[3], 'D MMMM YYYYha', 'Australia/Melbourne');
```

Comment:
@olafveerman do you think this should be h[2] instead of h[0] (end time vs start time)? I think normally when we get measurements, we know when they were put up, so we're keeping track of the last time that the measurement corresponded to.